controllers/cloud.redhat.com/proxy_client.go

```diff
@@ -21,21 +22,34 @@ type ProxyClient struct {
 	client.Client
 }
 
+func (p *ProxyClient) Get(ctx context.Context, key types.NamespacedName, obj runtime.Object) error {
+	clientOpsMetric.With(prometheus.Labels{"operation": "get"}).Inc()
```

Comment:
Are you okay knowing that this get will often return locally cached data?

Reply:
Yes

Reply:
It's just to track these things